formstack-baton-requests/cloud-formation.yaml

```diff
@@ -506,4 +568,212 @@ Resources:
+            "States": {
+              "UpdateDynamoInParallel": {
+                "Type": "Parallel",
+                "Next": "PerformSar",
+                "Branches": [
+                  {
+                    "StartAt": "AddAccountOneParam",
+                    "States": {
+                      "AddAccountOneParam": {
+                        "Type": "Pass",
+                        "Next": "UpdateDynamoWithAccountOne",
+                        "Result": "1",
+                        "ResultPath": "$.accountNumber"
+                      },
+                      "UpdateDynamoWithAccountOne": {
+                        "Type": "Task",
+                        "Resource": "${updateLambdaArn}",
+                        "Next": "CheckAccountOneUpdateCompletion"
+                      },
+                      "CheckAccountOneUpdateCompletion": {
+                        "Type": "Choice",
+                        "Choices": [
+                          {
+                            "Variable": "$.status",
+                            "StringEquals": "pending",
+                            "Next": "UpdateDynamoWithAccountOne"
+                          }
+                        ],
+                        "Default": "AccountOneUpdateComplete"
+                      },
+                      "AccountOneUpdateComplete": {
+                        "Type": "Pass",
+                        "End": true
+                      }
+                    }
+                  },
+                  {
+                    "StartAt": "AddAccountTwoParam",
+                    "States": {
+                      "AddAccountTwoParam": {
+                        "Type": "Pass",
+                        "Next": "UpdateDynamoWithAccountTwo",
+                        "Result": "2",
+                        "ResultPath": "$.accountNumber"
+                      },
+                      "UpdateDynamoWithAccountTwo": {
+                        "Type": "Task",
+                        "Resource": "${updateLambdaArn}",
+                        "Next": "CheckAccountTwoUpdateCompletion"
+                      },
+                      "CheckAccountTwoUpdateCompletion": {
+                        "Type": "Choice",
+                        "Choices": [
+                          {
+                            "Variable": "$.status",
+                            "StringEquals": "pending",
+                            "Next": "UpdateDynamoWithAccountTwo"
+                          }
+                        ],
+                        "Default": "AccountTwoUpdateComplete"
+                      },
+                      "AccountTwoUpdateComplete": {
+                        "Type": "Pass",
+                        "End": true
+                      }
+                    }
+                  }
+                ]
+              },
+              "PerformSar": {
+                "Type": "Task",
+                "InputPath": "$[0]",
```

Comment:
Only take the first result of the parallel branches as input. It doesn't matter which one it takes because it only uses initiationReference, subjectEmail and dataProvider in the perform lambda. These will be the same in both branch results.